CONTRIBUTING.md

```diff
@@ -29,13 +27,77 @@ and [this book’s style guide][tspl-style].
 [forum]: https://forums.swift.org/c/development/swift-docc/80
```

Comment:
Shall we update this to https://forums.swift.org/c/swift-documentation?

Reply:
Good catch — changed in 632d2b71.  This now matches the link in the GitHub issues template.